types/int_test.go

```diff
@@ -1,7 +1,6 @@
 package types
 
 import (
-	"math"
 	"math/big"
 	"math/rand"
```

Comment:
I would say, we should review every case where `math/rand` is used....
Maybe another issue to check this?

Reply:
as just discussed... maybe just remove the functionality to avoid possible future issues.

Reply:
the whole int_test.go will be gone once Ints are dropped